build/root/WORKSPACE

```diff
@@ -4,74 +4,53 @@ load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive", "http_file"
 load("//build:workspace_mirror.bzl", "mirror")
 
 http_archive(
-    name = "bazel_toolchains",
-    sha256 = "3a6ffe6dd91ee975f5d5bc5c50b34f58e3881dfac59a7b7aba3323bd8f8571a8",
-    strip_prefix = "bazel-toolchains-92dd8a7",
-    urls = [
-        "https://mirror.bazel.build/github.com/bazelbuild/bazel-toolchains/archive/92dd8a7.tar.gz",
-        "https://github.com/bazelbuild/bazel-toolchains/archive/92dd8a7.tar.gz",
-    ],
+    name = "io_bazel_rules_go",
+    sha256 = "142dd33e38b563605f0d20e89d9ef9eda0fc3cb539a14be1bdb1350de2eda659",
+    urls = mirror("https://github.com/bazelbuild/rules_go/releases/download/v0.22.2/rules_go-v0.22.2.tar.gz"),
 )
 
-load("@bazel_toolchains//rules:rbe_repo.bzl", "rbe_autoconfig")
+load("@io_bazel_rules_go//go:deps.bzl", "go_download_sdk", "go_register_toolchains", "go_rules_dependencies")
 
-rbe_autoconfig(
-    name = "rbe_default",
-    base_container_digest = "sha256:677c1317f14c6fd5eba2fd8ec645bfdc5119f64b3e5e944e13c89e0525cc8ad1",
-    digest = "sha256:b7c2e7a18968b9df2db43eda722c5ae592aafbf774ba2766074a9c96926743d8",
-    registry = "gcr.io",
-    repository = "k8s-testimages/bazel-krte",
-    # tag = "latest",
+go_rules_dependencies()
+
+go_register_toolchains(
+    go_version = "1.13.9",
 )
 
 http_archive(
-    name = "bazel_skylib",
-    sha256 = "eb5c57e4c12e68c0c20bc774bfbc60a568e800d025557bc4ea022c6479acc867",
-    strip_prefix = "bazel-skylib-0.6.0",
-    urls = mirror("https://github.com/bazelbuild/bazel-skylib/archive/0.6.0.tar.gz"),
+    name = "io_k8s_repo_infra",
+    sha256 = "a6ca952e365600a17f56f0fc8e41016e1d13cfb2b74c0c29bad6bdba3e3d8a4d",
+    strip_prefix = "repo-infra-0.0.3",
+    urls = mirror("https://github.com/kubernetes/repo-infra/archive/v0.0.3.tar.gz"),
 )
 
-load("@bazel_skylib//lib:versions.bzl", "versions")
+load("@io_k8s_repo_infra//:load.bzl", repo_infra_repositories = "repositories")
 
-versions.check(minimum_bazel_version = "0.23.0")
+repo_infra_repositories()
 
-http_archive(
-    name = "io_k8s_repo_infra",
-    sha256 = "f6d65480241ec0fd7a0d01f432938b97d7395aeb8eefbe859bb877c9b4eafa56",
-    strip_prefix = "repo-infra-9f4571ad7242bf3ec4b47365062498c2528f9a5f",
-    urls = mirror("https://github.com/kubernetes/repo-infra/archive/9f4571ad7242bf3ec4b47365062498c2528f9a5f.tar.gz"),
-)
+load("@io_k8s_repo_infra//:repos.bzl", repo_infra_go_repositories = "go_repositories", repo_infra_configure = "configure")
 
-http_archive(
-    name = "io_bazel_rules_go",
-    sha256 = "62bedd372f125fe62c16c0cc2ad9d7a2b6a1171d639933a5651a729fdce497fc",
-    urls = mirror("https://github.com/bazelbuild/rules_go/releases/download/v0.20.7/rules_go-v0.20.7.tar.gz"),
+repo_infra_configure(
+    minimum_bazel_version = "2.2.0",
+    rbe_name = None,
```

Comment:
Why?

Reply:
I don't want repo_infra_configure() to call rbe_autoconfig on me.

https://github.com/kubernetes/repo-infra/blob/8f6828353529bce756dc34bf80f5061612d7174c/repos.bzl#L27-L28

Before this change, we pull our own bazel_toolchain and don't call repo_infra_configure(). Above you recommended that we configure these dependencies using repo-infra. That's fine for the most part, but we explicitly pin an image for rbe.

Reply:
TLDR: This is dumb. We should fix repo_infra_configure().

Reply:
What is this pinned image doing?

Reply:
* repo-infra use the RBE default which is marketplace.gcr.io/google/rbe-ubuntu16-04.
* k/k has used a [variant](https://github.com/kubernetes/test-infra/blob/16cf2dd910060385e538098f4892d11f93327aaf/images/bazel-krte/Dockerfile) of marketplace.gcr.io/google/bazel up until this point that had rpm installed, but we don't need it anymore because we removed rpm rules from this repo.

No arguments against standardizing all CI on a single RBE image. We just have not done that to this point and it.

Reply:
Moving that change into its own PR sgtm

Reply:
I filed https://github.com/kubernetes/repo-infra/issues/182 to track